public/app/plugins/panel/geomap/style/markers.ts

```diff
@@ -206,7 +227,7 @@ export function getMarkerAsPath(shape?: string): string | undefined {
 }
 
 // Will prepare symbols as necessary
-export async function getMarkerMaker(symbol?: string): Promise<StyleMaker> {
+export async function getMarkerMaker(symbol?: string, text?: string): Promise<StyleMaker> {
```

Comment:
is text required here?  the text is actually passed to the maker so I think we can skip it here